fbw-common/src/wasm/systems/systems/src/navigation/adirs.rs

```diff
@@ -652,6 +652,22 @@ impl AirDataInertialReferenceUnit {
     fn altitude(&self) -> Arinc429Word<Length> {
         self.adr.altitude()
     }
+
+    fn ground_speed(&self) -> Arinc429Word<Velocity> {
+        self.ir.ground_speed()
+    }
+
+    fn true_airspeed(&self) -> Arinc429Word<Velocity> {
+        self.adr.true_airspeed()
+    }
+
+    fn baro_correction_1(&self) -> Arinc429Word<Pressure> {
+        self.adr.baro_correction_1()
+    }
+
+    fn ambient_static_pressure(&self) -> Arinc429Word<Pressure> {
+        self.adr.corrected_average_static_pressure()
+    }
```

Comment:
In reality the IR and ADR have their own separate ARINC busses, so combining them like this in an "ADIRU" bus is odd. We have already gone down that path though so probably not for this PR to fix.

Reply:
This was merged on master. I didn't add the function